scripts/gis-material-update/validate-deploy/modules/common.py

```diff
@@ -0,0 +1,91 @@
+import math
+from sqlalchemy import create_engine, text
+    
+def deploy(pg_conn_uri, tormays_table_org, tormays_table_temp, logger=None):
+    # validate data amount: is it between given limits
+    delete_result = None
+    engine = create_engine(pg_conn_uri)
+    with engine.connect() as connection:
+        transaction = connection.begin()
+        delete_sql = "delete from " + tormays_table_org
+        try:
+            logger.info("Deploy ...")
+            delete_result = connection.execute(text(delete_sql))
+            logger.info(tormays_table_org + ": " + f"{delete_result.rowcount} rows deleted.")
+        except Exception as e:
+            # Transaction implicitly rolls back if an exception occurred within the "with" block
+            logger.error(f"Error: {e}")
+        
+        if delete_result:    
+            copy_sql = "insert into " + tormays_table_org + " select * from " + tormays_table_temp
```

Comment:
Using `SELECT *` with `INSERT` is a bit dangerous, since there's a chance that the columns are in a different order, which could either fail or insert data to the wrong columns. E.g. `rush_hour` to `direction_id` and vice versa for `tormays_buses_polys`.

With delete and insert, there's also no way to change the columns of the target table. Dropping the table and recreating it would enable that.

But this might be something for future development.

Reply:
This part also changed so that tormays table data is replaced data from file and the tormays table is recreated always.